pkg/apis/autoscaling/annotation_validation.go

```diff
@@ -43,21 +43,49 @@ func ValidateAnnotations(annotations map[string]string) *apis.FieldError {
 		return nil
 	}
 
+	return validateMinMaxScale(annotations).Also(validateWindows(annotations))
+}
+
+func validateWindows(annotations map[string]string) *apis.FieldError {
+	var errs *apis.FieldError
+	if v, ok := annotations[PanicWindowPercentageAnnotationKey]; ok {
+		if fv, err := strconv.ParseFloat(v, 64); err != nil {
+			errs = errs.Also(apis.ErrInvalidValue(v, "annotation: "+PanicWindowPercentageAnnotationKey))
+		} else if fv < PanicWindowPercentageMin || fv > PanicWindowPercentageMax {
+			errs = apis.ErrOutOfBoundsValue(v, PanicWindowPercentageMin,
+				PanicWindowPercentageMax, "annotation: "+PanicWindowPercentageAnnotationKey)
+		}
+	}
+	if v, ok := annotations[PanicThresholdPercentageAnnotationKey]; ok {
+		if fv, err := strconv.ParseFloat(v, 64); err != nil {
+			errs = errs.Also(apis.ErrInvalidValue(v, "annotation: "+PanicThresholdPercentageAnnotationKey))
+		} else if fv < PanicThresholdPercentageMin {
+			errs = errs.Also(&apis.FieldError{
+				Message: fmt.Sprintf("Invalid annotation value %s, must be at least %v", v, PanicThresholdPercentageMin),
+				Paths:   []string{PanicThresholdPercentageAnnotationKey},
```

Comment:
```suggestion
				Paths:   []string{"annotation: "+PanicThresholdPercentageAnnotationKey},
```

(at this point, maybe factor into a `annotationPath(key string)` function and use throughout the file?)

Reply:
I thought of that. I was 50/50. But if you want, I'll tip to 51.